src/plugins/template/backend/ops/if.hpp

```diff
@@ -0,0 +1,169 @@
+// Copyright (C) 2018-2023 Intel Corporation
```

Comment:
we need to split cpp files into multiple per-operation cpp file. 
splitting up in header files will not help, because compilation time will not decrease

Reply:
> we need to split cpp files into multiple per-operation cpp file. splitting up in header files will not help, because compilation time will not decrease

Yes, I know, the next step will split the implementation from hpp to cpp.